### ISSUE TITLE
Download link SSL certificate expired

### ISSUE THREAD

**Issue by jarmo** [allinurl/goaccess#2190]
Starting from today (24.09.21) SSL certificate has been expired when trying to download GoAccess. Clicking on a download link will show that site is "dangerous".

Download link here for example https://tar.goaccess.io/goaccess-1.5.1.tar.gz

**Comment by allinurl** [comment]
Thanks for reporting this. I'm not seeing any expire certificates on my end. Do you know what was the exact message that popped up for you?

![2021-09-25-105241_520x627_scrot](https://user-images.githubusercontent.com/5005367/134777684-29d71e33-2b38-4175-8cf5-40896087a191.png)

**Comment by jarmo** [comment]
I cannot reproduce it either anymore. It was the regular message where certificate is not valid and upon inspection it expired on 24.09.21.

Is it possible that this download endpoint is behind some load-balancer having multiple servers?

**Comment by allinurl** [comment]
That's a good point. It's not behind a load-balancer. However, I did see some stalled bind processes and just restarted the server. I'll keep an eye on this as someone else reported a few months back something similar that appeared to be a DNS issue. Thanks again!

**Comment by Habetdin** [comment]
@allinurl I believe that `*.goaccess.io` certificate(s) should be renewed with the `--preferred-chain  "ISRG Root X1"` to exclude expired root from the trust chains.

**Comment by allinurl** [comment]
@Habetdin just curious, did you recently get an error?

**Comment by allinurl** [comment]
@Habetdin by the way, I just added `preferred_chain = ISRG Root X1` under `/etc/letsencrypt/renewal/*.goaccess.io`. Thanks for the heads up!

**Comment by Habetdin** [comment]
> @Habetdin just curious, did you recently get an error?

@allinurl yes, I've got same error as in [this post](https://github.com/allinurl/goaccess/issues/2190#issuecomment-940081065) when used `apt` on 7th of November.
Also, expired alternative trust chain breaks connection to custom DoT servers in Android and connection to some git servers that are using similar certificates. I've fixed issues like that from server side.

> Thanks for the heads up!

No problem, that's just the way to fix it without client-side actions required.

But somewhy I still see expired trust path for `deb.goaccess.io` while checking it at SSLLabs' Tester.

Seems like `preferred_chain` should be set for explicit subdomains in your case, cuz certbot treats `*.goaccess.io` as name of wildcard certificate but not as wildcard for subdomains like `deb.goaccess.io` themselves.
